provision/acc_provision/flavors.yaml

```diff
@@ -1193,6 +1193,133 @@ flavors:
     status: Experimental
     hidden: False
     order: 22
+  eks:
+    desc: Elastic Kubernetes Service (aws)
+    default_version: 5.2
+    overlay: True
+    config:
+      net_config:
+        max_csr_tunnels: 0
+        csr_list: ""
+        node_subnet: "172.0.0.1/24"
+      cloud:
+        provider: aws
+      kube_config:
+        no_priority_class: True
+        use_openshift_run_level: True
+        system_namespace: aci-containers-system
+        use_aci_anywhere_crd: True
+        pod_exec_resources_controller: True
+        skip_namespace_resource_hostagent: False
+        skip_replicationcontroller_resource_hostagent: False
+        all_access_controller_clusterrole: True
+        no_hostagent_networking_clusterrole: False
+        no_hostagent_deployment_clusterrole: False
+        run_opflex_server_container: True
+        run_gbp_container: True
+        use_opflex_server_volume: True
+        use_cnideploy_initcontainer: True
+        generate_cnet_file: True
+        opflex_mode: overlay
+      aci_config:
+        contracts:
+          - name: allow-all-out
+            underlay: True
+            filter: match-all
+            provided: ["inet-ext"]
+            consumed: ["ul-private", "ul-public"]
+          - name: allow-all-in
+            underlay: True
+            filter: match-all
+            consumed: ["inet-ext"]
+            provided: ["ul-private", "ul-public"]
+          - name: allow-cidr
+            underlay: True
+            filter: match-oshift
```

Comment:
Do you need to use this?

Reply:
Yes, this one I'll keep for now. The requirements are similar.